web/components/badges/badge.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Col } from '../layout/col'
+import { Row } from '../layout/row'
+
+export function Badge(props: { icon: React.ReactNode; label: string }) {
+  const { icon, label } = props
+  return (
+    <div className="margin-left-5 flex items-center">
+      <span
+        className={`inline-block whitespace-nowrap rounded-full bg-blue-600 py-2.5 px-2.5 text-center align-baseline text-xs font-bold leading-none text-white`}
```

Comment:
creates the placeholder badge shape in a solid blue color

Reply:
Sweet! This is a decent first pass on the badge design. I wasn't immediately able to guess which badge was for traders / creators (I had it backwards).

I'm not a designer, but I would remove the `#` and make the rank number larger. Maybe add a border. The blue background is a little strong.

Would be nice to contract out some icons, e.g. https://dribbble.com/shots/16325934-Gamification-Badges but that would come later. For the first version, I wouldn't worry too much :-)

Reply:
I'll remove the #, thanks!